src/ray/core_worker/transport/actor_task_submitter.cc

```diff
@@ -259,6 +259,10 @@ Status ActorTaskSubmitter::SubmitTask(TaskSpecification task_spec) {
 }
 
 void ActorTaskSubmitter::DisconnectRpcClient(ClientQueue &queue) {
+  if (queue.rpc_client) {
+    queue.max_finished_seq_no_start_at_ = queue.rpc_client->ClientProcessedUpToSeqno() +
+                                          queue.inflight_task_callbacks.size();
```

Comment:
Please add a comment to explain the formula

Reply:
resolved this comment because it is outdated.